monai/transforms/transforms.py

```diff
@@ -12,6 +12,7 @@
 
 import torch
 import numpy as np
+from skimage.transform import rotate
```

Comment:
skimage isn't listed as a dependency in requirements.txt, an import won't be possible if it's missing.

Reply:
Also, as briefly discussed in https://github.com/Project-MONAI/MONAI/issues/112#issuecomment-590517152, pytorch provides a handy `grid_sample` function which seems also much more efficient than skimage.transform (at least for generic warps in 2D). See https://colab.research.google.com/drive/17pI6xPGLTGcpGBulBrWZeNnBfLAwzqZT

Reply:
strangely the CI test passed without modifying `requirements.txt`, @IsaacYangSLA looks like the CI docker image has common packages preinstalled, could we have a minimal docker image instead?

Reply:
Updated to use scipy.ndimage. Scikit-image doesn't work for 3D images properly. Also added to requirements and test.

Reply:
@tvercaut That request is being handled in another PR.